webviews/src/App.tsx

```diff
@@ -11,10 +11,12 @@ function App() {
     { label: 'granite-code:3b', value: 'granite-code:3b', info: '2.0 GB' },
     { label: 'granite-code:8b', value: 'granite-code:8b', info: '4.6 GB' },
     { label: 'granite-code:20b', value: 'granite-code:20b', info: '12 GB' },
-    { label: 'granite-code:34b', value: 'granite-code:34b', info: '19 GB' }
+    { label: 'granite-code:34b', value: 'granite-code:34b', info: '19 GB' },
+    { label: 'Keep existing configuration', value: 'Keep existing configuration', info: '' }
   ];
   const embeddingsOptions: ModelOption[] = [
-    { label: 'nomic-embed-text', value: 'nomic-embed-text:latest', info: '274 MB' }
+    { label: 'nomic-embed-text', value: 'nomic-embed-text:latest', info: '274 MB' },
+    { label: 'Keep existing configuration', value: 'Keep existing configuration', info: '' }
```

Comment:
can we have a null value instead? That'd avoid having to touch the onChange handlers

Reply:
yes we can. But for onChange handler , we need to check 'Keep existing configuration' option and not to pass value to the config.JSON file.